src/components/pacbio/PacbioLibraryForm.vue

```diff
@@ -253,11 +210,31 @@ const provider = async () => {
 }
 
 provider()
-/**
- * @method resetSelectedTagId
- * Resets the selected tag ID to an empty string.
- */
+
 const resetSelectedTagId = () => {
   formLibrary.value.tag_id = ''
 }
+
+const libraryErrorsFor = (attribute) => {
+  if (formLibrary.value?.[attribute]?.length) {
+    delete formLibrary.value?.errors?.[attribute]
+    return ''
+  }
+  return formLibrary.value?.errors?.[attribute]
+}
+
+const inputBorderClass = (attribute) => {
+  return formLibrary.value[attribute] ? 'rounded-border' : 'rounded-border red-border'
+}
 </script>
+
+<style scoped>
+.rounded-border {
+  border-radius: 0.46rem;
+  border-width: 1px;
+}
+
+.red-border {
+  border-color: #f00;
+}
+</style>
```

Comment:
As part of tailwind migration, we have decided not to use inline styles. Please refactor the CSS to use Tailwind classes instead of custom styles which helps to maintain consistency and readability across the project.